public_src/components/toolbar/toolbar.component.ts

```diff
@@ -167,4 +181,20 @@ export class ToolbarComponent implements OnInit {
   private isDisabled(): boolean {
     return this.currentElement.id < 0;
   }
+
+  public toggleRouteInfoLabels(): void {
+   if (!this.routeLabelShown) {
+     this.mapSrv.showRouteInfoLabels(this.storageSrv.currentElement);
+     this.routeLabelShown = true;
+   }  else {
+     this.mapSrv.clearSingleRouteInfoLabels();
+     this.routeLabelShown = false;
+   }
+  }
+
+  public hasRef(): boolean {
+    if (this.storageSrv.currentElement.tags.ref) {
```

Comment:
Delete if because you cant return false here and simply use "return this.storage..."?